pkg/network/protocols/grpc/monitor_test.go

```diff
@@ -352,6 +352,9 @@ func (s *USMgRPCSuite) TestLargeBodiesGRPCScenarios() {
 	t.Cleanup(srv.Stop)
 	defaultCtx := context.Background()
 
+	longName1 := randStringRunes(30 * 1024 * 1024)
```

Comment:
Please prefer a better name and a comment explaining the optimization for future us

Reply:
changed